sources/world/world_countries_v2.hjson

```diff
@@ -39,6 +55,16 @@
       name: name
       desc: Country name
     }
+    {
+      type: property
```

Comment:
I think we're introducing a new data type here. I suggest `metric`. Otherwise, population and area will show up as join fields in Elastic Maps.

Reply:
What's the difference then between `id` and `property` types?

Reply:
`id` types should be used for join fields, they are usually ISO codes or other statistical codes. `property` types should be only used in tooltips, because they are usually country or region names (unsuitable for joining). However, for legacy reasons, Kibana still includes all field types in the join fields in Maps and the legacy viz.